scene/gui/text_edit.cpp

```diff
@@ -1229,6 +1229,58 @@ void TextEdit::backspace_at_cursor() {
 
 }
 
+void TextEdit::indent_selection_right() {
+
+	if (!is_selection_active()) {
+		return;
+	}
+	begin_complex_operation();
+
+		int start_line = get_selection_from_line();
```

Comment:
This block is wrongly indented. Is it because of the calls to _begin/end_complex_operation_? If so, I don't know if it fits our code standards (do we even have coding standards? :P ).